Matt-Hood/vons_fashsion
# Feature/stripe checkout payment

Description:
Configured checkout flows with the stripe test api 
Added user accounts for me, vince, and yvonee

Test Instructions:
Verify checkout flow works

Next steps:
Use config split or settings.php to alter the config to use to live stripe api on production
